guides/common/modules/proc_increasing-the-logging-levels-to-help-with-debugging.adoc

```diff
@@ -1,11 +1,11 @@
 [id="Increasing_Logging_Levels_to_Help_with_Debugging_{context}"]
-= Increasing Logging Levels to Help with Debugging
+= Increasing logging levels to help with debugging
 
 You can increase the logging level of {Project} components to troubleshoot {Project}.
 Increasing the logging level to `debug` provides the most detailed information.
 The default logging level is `info`.
 
-== Increasing the Logging Level for the Foreman Application
+== Increasing the logging level for the foreman application
```

Comment:
```suggestion
== Increasing the logging level for the Foreman application
```

Reply:
Oops. Didn't see the level 2 headers. Will fix them all.